src/ad-hoc-visualizations/color/visualization.tsx

```diff
@@ -1,18 +1,17 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License.
-import * as React from 'react';
-
+import { AdHocTestkeys } from 'common/configs/adhoc-test-keys';
+import { TestMode } from 'common/configs/test-mode';
+import { VisualizationConfiguration } from 'common/configs/visualization-configuration';
+import { Messages } from 'common/messages';
+import { TelemetryDataFactory } from 'common/telemetry-data-factory';
+import { VisualizationType } from 'common/types/visualization-type';
+import { generateUID } from 'common/uid-generator';
 import { adhoc as content } from 'content/adhoc';
-import { AdHocTestkeys } from '../../common/configs/adhoc-test-keys';
-import { TestMode } from '../../common/configs/test-mode';
-import { VisualizationConfiguration } from '../../common/configs/visualization-configuration';
-import { Messages } from '../../common/messages';
-import { TelemetryDataFactory } from '../../common/telemetry-data-factory';
-import { VisualizationType } from '../../common/types/visualization-type';
-import { generateUID } from '../../common/uid-generator';
-import { AdhocStaticTestView } from '../../DetailsView/components/adhoc-static-test-view';
-import { ScannerUtils } from '../../injected/scanner-utils';
-import { VisualizationInstanceProcessor } from '../../injected/visualization-instance-processor';
+import { AdhocStaticTestView } from 'DetailsView/components/adhoc-static-test-view';
+import { ScannerUtils } from 'injected/scanner-utils';
+import { VisualizationInstanceProcessor } from 'injected/visualization-instance-processor';
+import * as React from 'react';
```

Comment:
This is the only weird thing about this way of organizing our Imports. `react` is ending up at the end; its much easier to not forget importing `react` when its at top!

Reply:
I notice that too... but the imports are grouped base on relative/absolute path and then each group is alphabetized... not sure if we can change how tslint sort them.